.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.4/apache-maven-3.8.4-bin.zip
```

Comment:
We should update to 3.8.6, but not necessarily in this PR.

Reply:
Oh, I had 3.8.6 there, but something was not working so I have set 3.8.4. But the problem was unrelated, IIRC, so 3.8.6 is fine.

Reply:
Done https://github.com/quarkiverse/quarkus-cxf/pull/487/commits/83d1f7c36bdfca6f417f30ee42355e9356edc2e9#diff-1df564955cac198c2a317aceb01d86891160f1b185d9875b384b7d54819af358R17